.github/ISSUE_TEMPLATE/good-first-issue.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Good first issue
+about: Create an issue to welcome a new contributor into the community.
+title: ''
+labels: good-first-issue
+assignees: ''
+
+---
+
+## Background
+
+Overview of your issue here.
+
+## Instructions
+Hi, this is a `good-first-issue` issue. This means we've worked to make it more legible to people who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.
+
+We're interested in helping you take the first step, and can answer questions and help you out along the way. Note that we're especially interested in contributions from underrepresented groups!
+
+We know that creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+
+If you have contributed before, **consider leaving this PR for someone new**, and looking through our general [bug](https://github.com/ros-controls/ros2_control/labels/bug) issues. Thanks!
+
+### 🤔 What you will need to know.
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### 📋 Step by Step
+
+- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
+
+- [ ] 🗄️ **Create a local workspace** for making your changes and testing [following these instructions](https://moveit.ros.org/install/source/) with the exception replace  the "Download Source Code" section with [these instructions](https://ros-controls.github.io/control.ros.org/getting_started.html#compiling).
```

Comment:
Referencing ROS1 stuff here will be very confusing. I think we should
* only use ROS2 references
* have our own setup description to use here

Reply:
Should that be added to this file directly or should it be it's own section in control.ros.org?

Reply:
Could we remove this MoveIt reference? We should reference ros2 documentation.

Reply:
@bmagyar can you correct his? and then merge this PR

Reply:
```suggestion
- [ ] 🗄️ **Create a local workspace** for making your changes and testing [following these instructions](https://docs.ros.org/en/foxy/Tutorials/Workspace/Creating-A-Workspace.html), for Step3 use "Download Source Code" section with [these instructions](https://ros-controls.github.io/control.ros.org/getting_started.html#compiling).
```